src/components/LoadingMessage.css

```diff
@@ -1,15 +1,31 @@
 .loading-message {
-  height: 50px;
-  width: 100%;
-  border: 1px solid black;
   z-index: 1;
-  display: flex;
-  justify-content: center;
-  align-items: center;
+  display: inline;
+  /* justify-content: center;
```

Comment:
Looks like we can remove this commented code, yah?

Reply:
yes!

Reply:
Went through and got rid of comments